packages/novu/src/commands/init/templates/app-agent/ts/README-template.md

```
@@ -0,0 +1,69 @@
+
+A conversational AI agent powered by [Novu](https://novu.co) and [Next.js](https://nextjs.org).
+
+## Getting Started
+
+1. Start the development server:
+
+```bash
+npm run dev
+```
+
+2. Connect a chat platform in the [Novu Dashboard](https://dashboard.novu.co).
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)